src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionService.cs

```diff
@@ -126,14 +134,15 @@ ImmutableArray<CompletionItem> GetItemsFromCacheResult(GetCacheResult cacheResul
                     }
                     else
                     {
-                        // If there's cache miss, we just don't return any item.
+                        // If there's cache miss, we just don't return any item from this reference,
                         // This way, we will not block completion building our cache.
-                        return null;
+                        hasFullResults = false;
+                        continue;
```

Comment:
i'm a little confused in that it looks like there's some dupe in this PR.  i.e. we're checking for cache misses here... and also a few files below? not sure i get it.  want to chat?

Reply:
this change involves both type and ext method completion. Is that the dupe you are talking about?